src/utils/field-names.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * (c) Copyright Reserved EVRYTHNG Limited 2021. All rights reserved.
+ * Use of this material is subject to license.
+ * Copying and unauthorised use of this material strictly prohibited.
+ */
+
+/**
+ * An object containing all the possible field names for an attributeElement
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+export const attributeElement = {
+  id: 'id',
+  attribute: 'attribute',
+};
+
+/**
+ * An object containing all the possible field names for a bizLocation
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+export const bizLocation = {
+  id: 'id',
+};
+
+/**
+ * An object containing all the possible field names for a bizTransactionElement
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+export const bizTransactionElement = {
+  type: 'type',
+  bizTransaction: 'bizTransaction',
+};
+
+/**
+ * An object containing all the possible field names for a destinationElement
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+export const destinationElement = {
+  type: 'type',
+  destination: 'destination',
+};
+
+/**
+ * An object containing all the possible field names for an errorDeclaration
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+export const errorDeclaration = {
+  declarationTime: 'declarationTime',
+  reason: 'reason',
+  correctiveEventIDs: 'correctiveEventIDs',
+};
+
+/**
+ * An object containing all the possible field names for an ilmd
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+export const ilmd = {
+  type: 'type',
+  format: 'format',
+};
+
+/**
+ * An object containing all the possible field names for persistentDisposition
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+export const persistentDisposition = {
+  set: 'set',
+  unset: 'unset',
+};
+
+/**
+ * An object containing all the possible field names for a quantityElement
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+export const quantityElement = {
+  epcClass: 'epcClass',
+  quantity: 'quantity',
+  uom: 'uom',
+};
+/**
+ * An object containing all the possible field names for a readPoint
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+export const readPoint = {
```

Comment:
I think we need to change the content of this file a bit. It won't be clear to import something called 'readPoint' that contains all the possible fields of a readpoint (same for each parameter). You should call it something like 'readPointProperties'.

But I think what would be better is to create a new object, called 'properties' and to add fields to this object, i.e :
properties: {
  readpoint: {},
  event: {},
  ...
}